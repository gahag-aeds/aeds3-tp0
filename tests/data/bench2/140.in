140 70
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 67 12 22 57 71 86 52 93 94 11 64 11 40 35 83 6 52 71 65 23 70 69 24 12 9 1 75 75 26 31 43 45 95 17 2 19 3 54 12 98 17 28 9 9 63 44 15 67 67 33 90 37 54 66 2 63 67 77 91 93 60 86 90 8 3 92 27 58 98 39 56 67 67 18 76 82 14 44 49 82 29 91 71 83 57 73 47 25 2 90 70 63 76 61 71 79 5 50 37 56 89 46 23 8 16 52 42 30 48 91 12 77 34 84 12 91 9 59 68 12 
Sub 4 49
Max 86 131
Sub 16 61
Sum 54 99
Add 73 118
Min 53 98
Add 67 112
Max 88 133
Add 29 74
Sum 5 50
Sub 90 135
Max 18 63
Add 35 80
Sum 42 87
Sub 92 137
Min 71 116
Sub 78 123
Sum 26 71
Add 9 54
Min 18 63
Add 73 118
Min 50 95
Add 17 62
Min 53 98
Sub 20 65
Sum 65 110
Sub 16 61
Max 83 128
Add 89 134
Sum 51 96
Add 63 108
Sum 24 69
Sub 68 113
Min 78 123
Sub 78 123
Max 46 91
Sub 62 107
Min 76 121
Add 65 110
Max 90 135
Sub 12 57
Sum 87 132
Add 39 84
Sum 72 117
Sub 60 105
Sum 36 81
Sub 48 93
Min 43 88
Add 61 106
Min 57 102
Sub 60 105
Sum 65 110
Sub 12 57
Min 8 53
Add 23 68
Sum 57 102
Sub 4 49
Min 39 84
Add 45 90
Sum 92 137
Add 89 134
Max 14 59
Sub 48 93
Max 88 133
Add 23 68
Min 57 102
Add 39 84
Sum 17 62
Sub 38 83
Min 35 80
