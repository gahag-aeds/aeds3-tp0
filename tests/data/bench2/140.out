98
2223
1
100
2127
98
2148
2
2033
2
3
4
2685
100
2395
2025
2
100
1
91
2421
2708
2109
1
2
2629
0
2419
0
2346
96
99
2
2030
2
